src/circle.rs

```diff
@@ -103,7 +103,7 @@ impl Shape for Circle {
         } else {
             // This is empirically determined to fall within error tolerance.
             let n = (1.1163 * scaled_err).powf(1.0 / 6.0).ceil() as usize;
-            // Note: this isn't minimum error, but it is simple and we can easily
+            // Note: this isn't minimum error, but it is simple, and we can easily
```

Comment:
```suggestion
            // Note: this isn't minimum error, but it is simple and we can easily
```

this works well enough without the comma splice :)